builtin/credential/cert/path_certs.go

```diff
@@ -228,6 +234,25 @@ func (b *backend) pathCertWrite(ctx context.Context, req *logical.Request, d *fr
 		}
 	}
 
+	var parsedCIDRs []*sockaddr.SockAddrMarshaler
+	if boundCIDRListRaw, ok := d.GetOk("bound_cidrs"); ok {
+
+		var boundCIDRList []string
+		if boundCIDRs, ok := boundCIDRListRaw.([]string); ok {
+			boundCIDRList = boundCIDRs
+		} else if boundCIDRListStr, ok := boundCIDRListRaw.(string); ok {
```

Comment:
You shouldn't need to do this -- TypeCommaStringSlice will already split this for you.